src/components/PoolInfoCard.tsx

```diff
@@ -106,37 +101,35 @@ function PoolInfoCard({ data }: Props): ReactElement | null {
       </InfoItem>
       <InfoItem>
         <Typography>{`${t("totalLocked")}:`}</Typography>
-        <Typography variant="subtitle1">{`${formattedData.reserve}`}</Typography>
+        <Typography variant="subtitle1">{`$${formattedData.reserve}`}</Typography>
       </InfoItem>
       <InfoItem>
-        <Typography>{t("dailyVolume") + ": "}</Typography>
-        <Typography variant="subtitle1">{formattedData.volume}</Typography>
+        <Typography>{t("dailyVolume")}:</Typography>
+        <Typography variant="subtitle1">${formattedData.volume}</Typography>
       </InfoItem>
 
-      {console.log("aprs ==>", formattedData.aprs)}
       <Box display="flex" mt={3} flexWrap="wrap">
-        <InfoItem>
-          {Object.keys(formattedData.aprs).map((key) => {
-            const symbol = formattedData.aprs[key]?.symbol as string
-            const apr = formattedData.aprs[key]?.apr
-            return formattedData.aprs[key]?.apr.gt(Zero) ? (
-              <React.Fragment key={symbol}>
-                {symbol.includes("/") ? (
-                  <Tooltip title={symbol.replaceAll("/", "\n")}>
-                    <Typography sx={{ borderBottom: "1px dotted" }}>
-                      Reward APR:
-                    </Typography>
-                  </Tooltip>
-                ) : (
-                  <Typography>{symbol} APR: &nbsp;</Typography>
-                )}
-                <Typography variant="subtitle1">
-                  {apr && formatUnits(apr, 18)}
-                </Typography>
-              </React.Fragment>
-            ) : null
-          })}
-        </InfoItem>
+        {Object.keys(formattedData.aprs).map((key) => {
+          const symbol = formattedData.aprs[key]?.symbol as string
+          const apr = formattedData.aprs[key]?.apr
+          return formattedData.aprs[key]?.apr.gt(Zero) ? (
+            <InfoItem key={symbol}>
+              {symbol.includes("/") ? (
+                <Tooltip title={symbol.replaceAll("/", "\n")}>
+                  <Typography sx={{ borderBottom: "1px dotted" }}>
+                    Reward APR:
+                  </Typography>
+                </Tooltip>
+              ) : (
+                <Typography>{symbol} APR: &nbsp;</Typography>
```

Comment:
Does using template literals: ` ${} ` here with a space work so you don't have to use &nbsp;?

It'd probably be `${symbol} APR: ` but with the backticks.

Reply:
Yes, it would be more pretty clear!